style.css

```diff
@@ -0,0 +1,744 @@
+  padding-top: 5vh;
+}
+
+.feature-speaking-button {
+  width: 100%;
+  height: 7vh;
+  background-color: white;
+}
+
+/* Partners Section Mobile */
+.partners {
+  background-color: #272a31;
+  color: #fff;
+  padding: 2vh 1vw 8vh 1vw;
+}
+
+.partners-images {
+  display: flex;
+  flex-direction: column;
+  gap: 8vh;
+}
+
+.partners-images-1 {
+  display: flex;
+  justify-content: space-around;
+  padding-top: 7vh;
+}
+
+.partners-images-2 {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.alibaba {
+  width: 90px;
+  height: 90px;
+  color: silver;
+}
+
+/* Footer Section Mobile */
+.footer {
+  display: flex;
+  justify-content: space-evenly;
+  padding: 8vh 4vw;
+  font-size: 1em;
+  color: #272a31;
+  line-height: 140%;
+}
+
+.logo-footer {
+  width: 15vw;
+  height: 15vh;
+}
+
+.footer-logo {
+  display: flex;
+}
+
+.footer-logo-a {
+  display: flex;
+  text-decoration: none;
+  align-items: center;
+  color: #4a77e0;
+}
+
+.footer-logo-a h2 {
+  color: #272a31;
+}
+
+.fa-confluence {
+  color: #ec5242;
+}
+
+.footer-p {
+  padding-left: 6vw;
+}
+
+.desktop-p {
+  display: none;
+}
+
+/* ***********************************************************************************************************/
+
+/* AboutUs Section Mobile */
+
+/* Header Content Mobile AboutUs */
+.header-content-about {
+  background-image: url(images/mainbackground.png);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 3vh 3vw;
+}
+
+.header-content-about h5 {
+  color: #ec5242;
+  font-size: 20px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.header-content-about h1 {
+  color: #ec5242;
+  font-family: 'COCOGOOSE', sans-serif;
+  align-self: center;
+  font-size: 25px;
+  text-align: center;
+}
+
+.header-paragraph-about {
+  display: flex;
+  background-color: white;
+  border: 1px solid #d3d3d3;
+  padding: 0.1vh 2vw;
+  text-align: center;
+  line-height: 140%;
+}
+
+.headear-about-contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-top: 5vh;
+  line-height: 160%;
+}
+
+.headear-about-contact span {
+  text-align: center;
+}
+
+/* About-us Section Mobile AboutUs */
+.about-us {
+  display: flex;
+  flex-direction: column;
+  border-bottom: 1px solid #272a31;
+}
+
+.about-us-image {
+  display: flex;
+}
+
+.about-us-image img {
+  width: 100%;
+  height: auto;
+}
+
+.about-us-2 {
+  display: flex;
+  flex-direction: column;
+  padding: 2vh 3vw;
+  text-align: center;
+}
+
+.about-quest {
+  color: #ec5242;
+}
+
+/* Past Activities Section Mobile */
+.past-activities {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 5vh;
+  padding: 5vh 5vw;
+}
+
+.past-activities p {
+  text-align: center;
+}
+
+.past-activities-section-container {
+  display: flex;
+  flex-direction: column;
+  gap: 5vh;
+}
+
+.past-activities-section {
+  justify-content: center;
+  align-items: center;
+  height: 35vh;
+  background-image: url(images/pastactivity1.png) !important;
```

Comment:
- [ ] In following the [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Do%20not%20overuse%20!important%20rule%20in%20your%20CSS%20style%20definitions.) you seem to be overusing the `!important` tag. Kindly reduce the usage of `!important` tags to the barest minimum as it makes the stylesheet hard to maintain.

Reply:
This issue is well resolved! ✔️